assets/javascripts/discourse/templates/components/chat-message.hbs

```diff
@@ -151,7 +151,15 @@
 
           {{#if message.error}}
             <div class="chat-send-error">
-              {{message.error}}
+              {{#if (eq message.error "network_error")}}
```

Comment:
Im mixed on this check, but the alternative is adding some kind of error type, which would be more state to handle, not sure it's worth the complexity